jsf/multi-jsf-installer/src/main/resources/myfaces-injection-module.xml

```diff
@@ -22,7 +22,7 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="org.jboss.as.jsf-injection" slot="${jsf-impl-name}-${jsf-version}">
+<module xmlns="urn:jboss:module:1.8" name="org.jboss.as.jsf-injection" slot="${jsf-impl-name}-${jsf-version}">
```

Comment:
Dependencies on the "org.jboss.as.jsf" module and on the "org.jboss.weld.api" module should be added to the dependencies in this file as well.